src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void returnInfo() {
+        System.out.println("Figure: circle, area: " + Math.round(Math.PI * radius * radius)
+                + " sq.units, radius: " + radius + " units, color: " + getColor());
```

Comment:
The method name 'returnInfo' is a bit misleading since it doesn't return anything but prints information to the console. Consider renaming it to something like 'printInfo' to more accurately describe its behavior.